src/patternfly/components/Expandable/expandable.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../patternfly-imports";
+
+.pf-c-expandable {
+  // Expandable Toggle
+  --pf-c-expandable__toggle--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-expandable__toggle--Color: var(--pf-global--link--Color);
+
+  // Expandable Toggle Icon
+  --pf-c-expandable__toggle-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-expandable__toggle-icon--Color: var(--pf-global--Color--100);
+
+  // Expandable Expanded
+  --pf-c-expandable__content--MarginTop: var(--pf-global--spacer--xl);
+  --pf-c-expandable__content--Color: var(--pf-global--Color--100);
```

Comment:
can you add `.pf-c-expandable` to the list at the top of `_base.scss`? If you add that, I think you can remove this var & color property.

Reply:
Yes!